content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -303,6 +317,11 @@ for x, y in list:
 
 </LanguageSection>
 
+<Info>
+In order to efficiently sort arrays in a descending order, or in any custom order, you'll need to define your own **custom comparator**. This is covered in
```

Comment:
couldn't sorting and then reversing work for bronze?

Reply:
just for descending order

Reply:
that would be less efficient than sorting with a custom comp. But yeah, for descending order you could iterate in other ways.

Reply:
the text implies that this makes it impossible I think, so it would be beneficial to reword this

Reply:
^^ my thoughts exactly

Reply:
Ok I'll make the below changes?
```suggestion
In order to efficiently sort arrays in a custom order, you'll need to define your own **custom comparator**. For example, instead of reversing a sorted array to get a descending sorted array, you could use a custom comparator. This is covered in
```

Reply:
you don't "need to define your own custom comparator" but it does make the process a lot easier (for most problems), which is what I'm trying to get at. Something along the lines of: "The easiest method of sorting in a custom order is the custom comparator" should suffice.

Reply:
bump @Aadit-Ambadkar

Reply:
.

Reply:
Done